Proton/Sources/Renderer/RendererView.swift

```diff
@@ -197,7 +197,8 @@ extension RendererView: RichTextViewDelegate {
     func richTextView(_ richTextView: RichTextView, shouldHandle key: EditorKey, modifierFlags: UIKeyModifierFlags, at range: NSRange, handled: inout Bool) { }
 
     func richTextView(_ richTextView: RichTextView, didFinishLayout finished: Bool) {
-        readOnlyEditorView.relayoutAttachments()
+        guard finished else { return }
```

Comment:
Hope this guard doesn’t break anything, but it looked like an oversight to me.

Reply:
👍 Exactly what we have in `EditorView` https://github.com/rajdeep/proton/blob/master/Proton/Sources/Editor/EditorView.swift#L868